src/widgets/histogram/chart.js

```diff
@@ -586,7 +586,7 @@ module.exports = cdb.core.View.extend({
   _setupDimensions: function () {
     this._setupScales();
     this._setupRanges();
-    this._onWindowResize();
+    this.onWindowResize();
```

Comment:
Why have we changed this method without the `_`?

Reply:
Now this method is used outside chart.js, so it has become public